src/engine/Input/Pointer.ts

```diff
@@ -236,7 +240,7 @@ export class Pointers extends Class {
          var x: number = e.pageX - Util.getPosition(this._engine.canvas).x;
          var y: number = e.pageY - Util.getPosition(this._engine.canvas).y;
          var transformedPoint = this._engine.screenToWorldCoordinates(new Vector(x, y));
-         var pe = new PointerEvent(transformedPoint.x, transformedPoint.y, 0, PointerType.Mouse, e.button, e);
+         var pe = new PointerEvent(transformedPoint.x, transformedPoint.y, e.pageX, e.pageY, 0, PointerType.Mouse, e.button, e);
```

Comment:
There's not a standard event that would expose that, we would need to calculate it manually (find offset of Canvas element)--which might be a good feature to add. The current functionality is what I would expect screenX and screenY to return, I would not expect it to be relative to the canvas element.